operator/v1/types_ingress.go

```diff
@@ -121,6 +121,22 @@ type IngressControllerSpec struct {
 	//
 	// +optional
 	NodePlacement *NodePlacement `json:"nodePlacement,omitempty"`
+
+	// bindCiphers is used to specify the cipher algorithms that are negotiated
+	// during the SSL/TLS handshake with an IngressController.
+	//
+	// If unset, the CipherProfileIntermediateType profile is used.
+	//
+	// +optional
+	BindCiphers *Ciphers `json:"bindCiphers,omitempty"`
```

Comment:
How about `bindCiphers`→`allowedCiphers` (and `bindSecurityProtocol`→`allowedSecurityProtocols` below)?

Reply:
In my latest commit, I have removed both top-level fields for `SecuritySpec`, which defines `Profile` and `CustomSettings` for managing connection security for an `IngressController`. I'm looking forward to hear what you think of the latest changes. IMO this provides a clear separation between the 2 approaches and additional settings can be added to `CustomSettings` as the need arises.